messages/src/tests/upload.rs

```diff
@@ -314,3 +316,12 @@ fn roundtrip_input_share_aad() {
         ),
     )])
 }
+
+#[test]
+fn upload_response_size() {
```

Comment:
We should add `roundtrip_encoding()` tests for `UploadRequest` and `UploadResponse` for completeness. That helper function will check the `encoded_len()` hint correctness as well.

Reply:
yeaaah, I didn't do it because `roundtrip_encoding` doesn't handle the new ParameterizedDecode types. I'll figure it out...